code/modules/mob/living/carbon/init_signals.dm

```diff
@@ -2,12 +2,23 @@
 /mob/living/carbon/register_init_signals()
 	. = ..()
 
+	RegisterSignal(src, SIGNAL_ADDTRAIT(TRAIT_AGENDER), PROC_REF(on_nobreath_trait_gain))
```

Comment:
```suggestion
	RegisterSignal(src, SIGNAL_ADDTRAIT(TRAIT_AGENDER), PROC_REF(on_agender_trait_gain))
```
Also, doesn't this need a trait loos counterpart too?

Reply:
I'll implement it soon, thanks Ghom